src/ce/crt0.src

```diff
@@ -105,6 +105,47 @@ launcher:
 .webaddress:
 	db	"http://tiny.cc/clibs",0
 
+
+	section .start
+
+mpTmrRange    := 0F20000h
+
+tmr2Counter   := 10h
+tmrCtrl       := 30h
+
+bTmr2Enable   := 3
+bTmr2Crystal  := 4
+bTmr2Overflow := 5
+bTmr2CountUp  := 10
+
+	public	___init_clock
```

Comment:
should this really be public? also it should be in a defined section

Reply:
is this code removed if the user isn't using clock() and related functions?

Reply:
Yes, it needs to be public. Referencing this symbol (or not) is what determined whether the section it's in (and it is in one) will be included.

Reply:
Oh that makes sense.

Reply:
I just double-checked. Yes, if none of the new timing functions are referenced, this initialization code is not included.